extensions/wikia/Recirculation/services/ParselyDataService.class.php

```diff
@@ -82,7 +83,7 @@ private function apiRequest( $type, $meta, $count ) {
 		}
 
 		$url = $this->buildUrl( $endpoint, $options );
-		$data = ExternalHttp::get( $url );
+		$data = ExternalHttp::get( $url, self::PARSELY_API_TIMEOUT );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)